Exercise4/visitorDemo.java

```diff
@@ -1,17 +1,17 @@
 package Exercise4;
 import java.util.*;
-public class visitorDemo {
+public class VisitorDemo {
 	public static void main(String[] args) throws DogExceptions {
 		Scanner sc=new Scanner(System.in);
 		Child c=new Child();
 		Dog d=new Dog();
 		System.out.println("Enter an item: ");
 		String s=sc.next();
-		c.playWithDog(s, d);
+		c.PlayWithDog(s, d);
```

Comment:
Method names should follow camelCasing. Eg: `calculateAge`, `playWithDog`
Class names should follow PascalCasing. Eg: `GamingComputer`, `VisitorDemo`